### PR TITLE
chore: Update master to 23.2-SNAPSHOT

### PR DESCRIPTION
**Pull request by mshabarov**
Since 23.1.0.beta1 has been release, it's time to update master.

### CODE REVIEW

File: flow-tests/vaadin-spring-tests/pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <properties>
         <maven.deploy.skip>true</maven.deploy.skip>
         <maven.failsafe.plugin.version>3.0.0-M1</maven.failsafe.plugin.version>
-        <vaadin.version>23.1-SNAPSHOT</vaadin.version>
```
Comment (ZheSun88) [inline flow-tests/vaadin-spring-tests/pom.xml:19]: let us keep this version for now.. after we have the platform 23.2-snapshot, we then update it

Comment (ZheSun88) [reply flow-tests/vaadin-spring-tests/pom.xml]: i made a commit to this PR.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Since [Vaadin platform 23.0](https://github.com/vaadin/platform), Flow major and
 |--------|-----------------------------------------------------------------|---------------------------------------------------------|
 |  1.0   |  10 (LTS)                                                       |  1.0                                                    |
 |  2.7   |  14.8.x (LTS)                                                   |  2.7.x                                                  |
-|  9.0   |  22.0.x                                                         |  9.x                                                    |
-|  23.0  |  23.0.x (LATEST)                                                |  23.0                                                   |
-|  master|  23.1 (NEXT)                                                    |  23.1                                                   |
+|  9.0   |  22.0.x (LATEST with Java 8+ support)                           |  9.x                                                    |
+|  23.0  |  23.0.x (LATEST with Java 11+ support)                          |  23.0.x                                                 |
```
Comment (mcollovati) [inline README.md:22]:
```suggestion
|  23.0  |  23.0.x (FIRST with Java 11+ support)                          |  23.0.x                                                 |
```

**Review by mcollovati — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mshabarov) [reply README.md:22]: Not sure I understand this change. What does it mean FIRST? I aligned the readme with this page https://vaadin.com/releases

**Review by mcollovati — APPROVED** [review]

**Pull request merged**